docs/features/authentication.md

```diff
@@ -1,17 +1,39 @@
 # Authentication
-Authentication is an enchanced Identity feature of mxisd to ensure coherent and centralized identity management.
+
+- [Description](#description)
+- [Overview](#overview)
+- [Getting started](#getting-started)
+  - [Synapse](#synapse)
+  - [mxisd](#mxisd)
+  - [Validate](#validate)
+- [Next steps](#next-steps)
+  - [Profile auto-fil](#profile-auto-fill)
+- [Advanced Authentication](#advanced-authentication)
+  - [Requirements](#requirements)
+  - [Configuration](#configuration)
+    - [Reverse Proxy](#reverse-proxy)
+      - [Apache2](#apache2)
+    - [DNS Overwrite](#dns-overwrite)
+    - [Backends](#backends) 
+
+## Description
+Authentication is an enhanced Identity feature of mxisd to ensure coherent and centralized identity management.
 
 It allows to use Identity stores configured in mxisd to authenticate users on your Homeserver.
 
+This feature can also provide the ability to users to login on the Homeserver using their thirdparty identities provided by this Identity store.
```

Comment:
- `hirdparty identities` is not used anymore else. Let's be consistent throughout the whole documentation. I believe I called it `3PIDs` everywhere else.
- `by this Identity store` - which "this"? There is no reference to an identity store in the sentence or paragraph.